### PR TITLE
Added support for `ToString()` method

### LINKED ISSUE #16: Nullable context is not covered.

### PR DESCRIPTION
**Pull request by lukaszkrzywizna**
closes #16 #3

### CODE REVIEW

File: Gundi/tests/Gundi.Tests/Gundi.Tests.csproj [hunk 1]
```diff
@@ -8,11 +8,16 @@
         <GenerateProgramFile>false</GenerateProgramFile>
         <WarningsAsErrors>Nullable</WarningsAsErrors>
         <IsPackable>false</IsPackable>
-        <!-- Uncomment to see generated files -->
```
Comment (BartoszKubica) [inline Gundi/tests/Gundi.Tests/Gundi.Tests.csproj:11]: Are you sure you want to leave it uncommented ?

File: Gundi/tests/Gundi.Tests/Generated/Gundi/Gundi.UnionGenerator/CSharpUnion.g.cs [hunk 2]
```diff
@@ -0,0 +1,246 @@
+﻿#nullable enable 
+
+using System;
```
Comment (BartoszKubica) [inline Gundi/tests/Gundi.Tests/Generated/Gundi/Gundi.UnionGenerator/CSharpUnion.g.cs:3]: Is it necessary to commit generated code ?

**Review by BartoszKubica — APPROVED** [review]

File: Gundi/tests/Gundi.Tests/Gundi.Tests.csproj (hunk 1 above)
Comment (lukaszkrzywizna) [reply Gundi/tests/Gundi.Tests/Gundi.Tests.csproj:11]: Yes, but I should uncomment the section about excluding files from the compilator. We can later decide if we want to track changes in generated files.

File: Gundi/tests/Gundi.Tests/Generated/Gundi/Gundi.UnionGenerator/CSharpUnion.g.cs (hunk 2 above)
Comment (lukaszkrzywizna) [reply Gundi/tests/Gundi.Tests/Generated/Gundi/Gundi.UnionGenerator/CSharpUnion.g.cs:3]: According to https://andrewlock.net/creating-a-source-generator-part-6-saving-source-generator-output-in-source-control/, I think we can track those files. Whenever we change something in the generator, we will be able to determine what has changed.

**Pull request merged**